File: src/main/kotlin/com/hoi4utils/clausewitz_parser/ParserException.kt
package main.kotlin.com.hoi4utils.clausewitz_parser

class ParserException : Exception {
    constructor() : super()
    constructor(message: String?) : super(message)
    constructor(message: String?, cause: Throwable?) : super(message, cause)
    constructor(cause: Throwable?) : super(cause)
}

File: src/main/scala/com/hoi4utils/clausewitz_parser/ParserException.scala
package com.hoi4utils.clausewitz_parser

class ParserException(message: String, cause: Throwable) extends Exception(message, cause) {
  def this(message: String) {
    this(message, null)
  }
  
  def this(cause: Throwable) {
    this(null, cause)
  }
}
